tests/materialdata/woehler/test_analyzer.py

```diff
@@ -223,10 +223,10 @@ def test_woehler_elementary_initialize_with_determined_fractures():
 def test_woehler_elementary_initialize_with_pandas_dataframe():
     expected = pd.Series({
         'SD': 362.5,
-        'k_1': 7.0,
-        'ND': 3e5,
-        'TN': 5.3,
-        'TS': 1.27,
+        'k_1': 9.88,
+        'ND': 417772,
+        'TN': 6.78,
+        'TS': 1.21,
```

Comment:
Seems a little bit strange to me, that the `k_1` and `ND` values increase. I would expect them to decrease as we no don't take the late fractures into account. I would expect the slope to be steeper.

Have you checked if that makes sense?

Reply:
Thanks for your comment and good observation. At the first look it does not to seem so make any sense, but it is due to the used data. The scattering of the fractures of the load levels in the infinite region are lower then the ones in the finite region causing this affect. I plotted both results and add it here, hoping this will make things clearer.

Reply:
Ok, fine. Are you finished with this. If yes then please press "Ready for review"